### ISSUE TITLE
ENTSO-E parser is failing for GB-NIR price data

### ISSUE THREAD

**Issue by madsnedergaard** [electricitymaps/electricitymaps-contrib#6352]
## Description

The parser is calling this URL, but it is always returning 400:
`https://web-api.tp.entsoe.eu/api?documentType=A44&in_Domain=10Y1001A1001A016&out_Domain=10Y1001A1001A016&periodStart=202401132000&periodEnd=202401162000&securityToken=TOKEN`

- `documentType=A44` = price
- 10Y1001A1001A016` = `GB-NIR`

Response received:

```
 <Reason>
                <code>999</code>
                <text>Delivered Area EIC 10Y1001A1001A016 is not valid.</text>
        </Reason>
```

**Comment by madsnedergaard** [comment]
cc @VIKTORVAV99, I was a bit unsure how to approach this, what do you think we should do? :)

**Comment by VIKTORVAV99** [comment]
Hmm, I'll double check but it might be that GB-NIR is not actually a valid zone for ENTSO-E. I'll take a look at it tomorrow.

**Comment by madsnedergaard** [comment]
Seems like it's only for price that the API calls are failing, but thanks for looking into it :)

**Comment by VIKTORVAV99** [comment]
> Seems like it's only for price that the API calls are failing, but thanks for looking into it :)

Yeah I think that is because they are having the same price info as with IE, so we might just have to override it to IE(SEM) instead of GB-NIR. I have not been able to confirm this just yet though.
